stravalib/model.py

```diff
@@ -759,20 +1043,23 @@ class Activity(
     end_latlng: Optional[LatLon] = None
     map: Optional[Map] = None
     gear: Optional[Gear] = None
-    best_efforts: Optional[List[BestEffort]] = None
-    segment_efforts: Optional[List[SegmentEffort]] = None
-    splits_metric: Optional[List[Split]] = None
-    splits_standard: Optional[List[Split]] = None
+    best_efforts: Optional[List[DetailedSegmentEffort]] = None
+    segment_efforts: Optional[List[DetailedSegmentEffort]] = None
+    # Ignoring types here given there are overrides
+    splits_metric: Optional[List[Split]] = None  # type: ignore
+    splits_standard: Optional[List[Split]] = None  # type: ignore
```

Comment:
I think it's good the be a bit more precise in the ignore comment and specify the actual error that we want to ignore (here `# type: ignore[assignment]`). Then if a warning for another rule is raised we will still get warned.

Reply:
cool - thanks. i wasn't paying attention closely to the fact that ignores had a specific type of error assigned.